users/templates/users/password_reset_form.html

```diff
@@ -0,0 +1,38 @@
+{% extends 'base.html' %}
+{% load crispy_forms_tags %}
+
+{% block content%}
+    <div class="container text-center mt-2 container-reset">
+        <h1>Password Reset Form</h1>
+        <p style="width: 60%; margin: 0 auto">
+            Please enter your new password so we can verify.
+        </p>
+        <br />
+        <form action="" method="POST">
+            {% csrf_token %}
+            {% if form.errors %}
+                {% for field in form %}
+                    {% for error in field.errors %}
+                        <div class="alert alert-danger">
+                            <strong>{{ error|escape }}</strong>
+                        </div>
+                    {% endfor %}
+                {% endfor %}
+            {% endif %}
+            {% for field in form %}
+                <div class="fieldWrapper mt-2 mb-2">
+                    {{ field.label_tag }} {{ field }}
+                </div>
+                 {% if field.name == 'new_password1' %}
+                        <small class="form-text text-muted">
+                            Your password can't be too similar to your other personal information.<br>
```

Comment:
Why hardcoded? It should be provided by Django, if not, better to remove, cause ift may change in future version and it will create confusion.